lib/tasks/temp.rake

```diff
@@ -1,6 +1,15 @@
 # -*- coding: utf-8 -*-
 namespace :temp do
 
+  desc 'Update EventType when only editing prominence to hide'
```

Comment:
@lizconlan I'm interested in why you've moved this to a rake task? Is it a convention we should be following for future contributions, or something specific to what this does?